api.md

```diff
@@ -171,7 +172,115 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_e
 ```
 
 ---
-## attach_tablespace() [](attach_tablespace)
+### add_data_node() [](add_data_node)
+
+Create a new data node for use when creating distributed hypertables.
```

Comment:
I would avoid the "create" terminology here since people might get the wrong idea of what this function does. I would say something like:

"Add a new data node to the current database for use with distributed hypertables. This includes bootstrapping the data node by creating and configuring a new database on it. Newly created distributed hypertables will be able to make use of this data node for distributing data across the cluster. Note, however, that already existing distributed hypertables may require a separate `attach_data_node()` call to schedule the data node for use with those tables as they do not automatically expand to include the new data node."